ghproxy/ghproxy.go

```diff
@@ -131,10 +131,8 @@ func main() {
 		cache = ghcache.NewDiskCache(http.DefaultTransport, o.dir, o.sizeGB, o.maxConcurrency)
 	}
 
-	if o.pushGateway != "" {
-		go metrics.PushMetrics("ghproxy", o.pushGateway, o.pushGatewayInterval)
-		go diskMonitor(o.pushGatewayInterval, o.dir)
-	}
+	go diskMonitor(o.pushGatewayInterval, o.dir)
```

Comment:
This should really be in the `else` block of the `if o.dir == "" ` conditional above.